src/sentry/static/sentry/app/components/events/contexts/trace/getTraceKnownDataDetails.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import moment from 'moment-timezone';
+
+import {Event, Organization} from 'app/types';
+import {t} from 'app/locale';
+import Button from 'app/components/button';
+import space from 'app/styles/space';
+import EventView from 'app/utils/discover/eventView';
+import {getTraceDateTimeRange} from 'app/components/events/interfaces/spans/utils';
+import {transactionSummaryRouteWithQuery} from 'app/views/performance/transactionSummary/utils';
+
+import {TraceKnownData, TraceKnownDataType} from './types';
+
+type Output = {
+  subject: string;
+  value: React.ReactNode | string | null;
+};
+
+function getUserKnownDataDetails(
+  data: TraceKnownData,
+  type: TraceKnownDataType,
+  event: Event,
+  organization: Organization
+): Output | undefined {
+  switch (type) {
+    case TraceKnownDataType.TRACE_ID: {
+      const traceId = data.trace_id || '';
```

Comment:
are you sure about `|| ''` ? I would use defined here, because it can be also 0 right?
```suggestion
      const traceId = data.trace_id
      
      if (!defined(traceId)) {....
```

Reply:
`data.trace_id` is a string.